From 07bad858d8a9b94fbc40d7537b38906b66bbf3e7 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Tue, 11 Apr 2023 00:19:05 +0200
Subject: [PATCH] ARM: dts: stm32: enable arm-wdt on STM32MP1 boards

Enable the arm_wdt on STM32MP1x boards. Set default timeout
to 32 seconds.

Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Iea2b010955f68a5f1cce146e06c0ccf4b14b247c
---
 arch/arm/boot/dts/st/stm32mp135f-dk.dts   | 5 +++++
 arch/arm/boot/dts/st/stm32mp157a-ed1.dts  | 5 +++++
 arch/arm/boot/dts/st/stm32mp157c-ed1.dts  | 5 +++++
 arch/arm/boot/dts/st/stm32mp157d-ed1.dts  | 5 +++++
 arch/arm/boot/dts/st/stm32mp157f-ed1.dts  | 5 +++++
 arch/arm/boot/dts/st/stm32mp15xx-dkx.dtsi | 5 +++++
 6 files changed, 30 insertions(+)

--- a/arch/arm/boot/dts/st/stm32mp135f-dk.dts
+++ b/arch/arm/boot/dts/st/stm32mp135f-dk.dts
@@ -93,6 +93,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &i2c1 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&i2c1_pins_a>;
--- a/arch/arm/boot/dts/st/stm32mp157a-ed1.dts
+++ b/arch/arm/boot/dts/st/stm32mp157a-ed1.dts
@@ -121,6 +121,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/st/stm32mp157c-ed1.dts
+++ b/arch/arm/boot/dts/st/stm32mp157c-ed1.dts
@@ -121,6 +121,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/st/stm32mp157d-ed1.dts
+++ b/arch/arm/boot/dts/st/stm32mp157d-ed1.dts
@@ -121,6 +121,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/st/stm32mp157f-ed1.dts
+++ b/arch/arm/boot/dts/st/stm32mp157f-ed1.dts
@@ -121,6 +121,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/st/stm32mp15xx-dkx.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp15xx-dkx.dtsi
@@ -129,6 +129,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &cec {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&cec_pins_b>;
